scripts/instantiate_bridge.ts

```diff
@@ -85,14 +85,20 @@ export const instantiateBridge = (
     )
   }
 
-  if (!l1Network.rpcURL) {
-    throw new Error('L1 rpc url not set (see .env.sample or networks.ts)')
-  }
-  if (!l2Network.rpcURL) {
-    throw new Error('L2 rpc url not set (see .env.sample or utils/networks.ts)')
-  }
-  const ethProvider = new JsonRpcProvider(l1Network.rpcURL)
-  const arbProvider = new JsonRpcProvider(l2Network.rpcURL)
+  const l1Rpc = (() => {
+    if (networkID === 42161) return process.env['MAINNET_RPC'] as string
+    if (networkID === 421611) return process.env['RINKEBY_RPC'] as string
+    if (networkID === 1338) return 'http://127.0.0.1:8545/'
```

Comment:
What will we do for unknown networks, like custom networks, local networks on different chain id or newly created devnets?

Reply:
they can have their own `instantiate_bridge` script or we can add support when needed.
this script isn't published as part of the package, its a dev utility